src/index.ts

```diff
@@ -45,37 +79,33 @@ function main(state: State, env: DotEnv) {
 			}
 		}
 
-		if (message.content.startsWith(config.commandPrefix)) {
-			// Handle commands.
-			// TODO: https://github.com/RSA-Bots/PandaProtector/issues/3
-			const content = message.content.slice(config.commandPrefix.length);
-			const matches = /^(\w+)\s*(.*)/su.exec(content);
-			const commandName = matches?.[1] ?? "";
-			const argumentContent = matches?.[2] ?? "";
-
-			if (commandName.length > 0) {
-				const command = getCommand(commandName);
-
-				if (command && command.hasPermission(state, message)) {
-					const args = command.parseArguments(argumentContent);
-					const required = command.options.reduce((acc, option) => acc + (option.optional ? 0 : 1), 0);
-
-					if (args.length >= required) {
-						command.handler(state, message, ...command.parseArguments(argumentContent));
-					} else {
-						ephemeral(state, message.reply(`Missing arguments for **${commandName}**.`)).catch(
-							console.error
-						);
-					}
-				}
+		// Handle meta commands.
+		if (message.member?.roles.cache.has(state.config.developerRoleId)) {
```

Comment:
Are we going to convert the to slash commands, and have them removed from the unload payload? (To remove hardcoded commands)

Reply:
Yeah that's probably a good idea.

Reply:
#60